pagefaultgames/pokerogue
# feat: added italian translation for the biomes

## What are the changes?
I saw that the biomes weren't localized in Italian, so here they are.  
Feel free to comment if you find a translation not accurate.

## Why am I doing these changes?
This PR aims to translate the selecatable biomes in Italian.
Also added some JSDoc to better identify and documenting the code.

### Screenshots/Videos
City:  
![Screenshot from 2024-05-29 23-23-48](https://github.com/pagefaultgames/pokerogue/assets/99277900/4f70f584-d52c-41a3-abb3-08d73cb63df0)  

Meadow:  
![Screenshot from 2024-05-29 23-25-19](https://github.com/pagefaultgames/pokerogue/assets/99277900/f2c97025-7ef9-4fcf-afd7-d29548f1b510)  

Desert:  
![Screenshot from 2024-05-29 23-25-47](https://github.com/pagefaultgames/pokerogue/assets/99277900/51a41cd5-8e29-48db-a36a-bb6b9e7075a2)  

Power Plant:  
![Screenshot from 2024-05-29 23-26-24](https://github.com/pagefaultgames/pokerogue/assets/99277900/97127adb-db5c-4dd3-a36a-45d8c566fa2b)


## How to test the changes?
Set the language in Italian and go through the gameplay, get a map and you can see the translations  

## Checklist
  - #1573 







Author: SavGRY - @SavGRY
